Assets/Scripts/Localization/LocalizationKeys.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PotentialRobot.Localization
+{
+    [CreateAssetMenu(menuName = "Localization/Localization Keys", fileName = "LocalizationKeys.asset")]
+    public class LocalizationKeys : ScriptableObject
+    {
+        [field: SerializeField]
+        public List<string> Keys { get; private set; }
+    }
```

Comment:
What is this asset used for? Why is this needed when the LocalizationAsset class exists?

Reply:
Used later for the editor to cross reference all the language assets whether they have the required keys or not. Not really needed for runtime.